app/models/katello/repository.rb

```diff
@@ -495,6 +495,24 @@ def ostree_branch_names
       self.ostree_branches.map(&:name)
     end
 
+    def update_ostree_branches!(branch_names)
+      non_uniq = branch_names.select { |e| branch_names.count(e) > 1 }.uniq
```

Comment:
Yeah that lists the non unique variables in that array.  That uniq call is important because it will return [1,1,2,2] for the select.